src/main/java/nextstep/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.qna.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DeleteHistories {
+    private List<DeleteHistory> deleteHistories = new ArrayList<>();
+
+    public void addHistory(DeleteHistory deleteHistories) {
+        this.deleteHistories.add(deleteHistories);
+    }
+
+    public void addALLHistory(DeleteHistories deleteHistories) {
```

Comment:
```suggestion
    public void addALLHistory(List<DeleteHistory> deleteHistorie) {
```

일급 콜렉션보다 콜렉션을 추가 가능하도록 지원하는 것은 어떨까?